src/blocks/CaseStudyParallax/index.module.scss

```diff
@@ -0,0 +1,314 @@
+@use '@scss/common' as *;
+
+.cards {
+  position: relative;
+}
+
+.mainGutter {
+  position: relative;
+
+}
+
+.card {
+  position: relative;
+  padding-top:  calc(var(--base) * 4);
+
+  &.isFirst {
+    grid-column: 1/16;
+    grid-row: 1/1;
+  }
+
+  & .media {
+    margin-right: calc(var(--gutter-h) / -1);
+    overflow: hidden;
+
+    img {
+      right: -10px;
+
+      @include mid-break {
+        right: unset;
+      }
+    }
+
+     @include mid-break {
+      margin-right: 0;
+     }
+  }
+
+  &:last-of-type {
+    padding-bottom: calc(var(--base) * 4);
+
+    @include mid-break {
+    padding-bottom:  calc(var(--base) * 0);
+  }
+  }
+}
+
+.stickyBlock {
+  position: sticky;
+  top: 18vh;
```

Comment:
@paulpopus Can we use a different `top` value here? Would need to test this - but maybe even `25%` would work, not sure

Same goes for the different breakpoints